user-impl/src/main/java/com/example/auction/user/impl/UserEntity.java

```diff
@@ -44,7 +44,7 @@ private Behavior notCreated() {
         );
 
         b.setCommandHandler(CreateUser.class, (create, ctx) -> {
-            User user = new User(UUID.fromString(entityId()), create.getName());
+            User user = new User(UUID.fromString(entityId()), create.getName(),"admin","admin");
```

Comment:
What I would do here is have the username as a property of user, but have the password as its own event, so you'd persist two events here, a `UserCreated` event, followed by a `PasswordUpdated` event.

In addition the `PasswordUpdated` event should store the password as a hash.  Bcrypt is good for hashing passwords:

http://www.mindrot.org/projects/jBCrypt/